test/screenshot/infra/lib/report-builder.js

```diff
@@ -996,20 +996,7 @@ class ReportBuilder {
   logRunParameters_(verb, screenshots) {
     const count = screenshots.length;
     const plural = count === 1 ? '' : 's:';
-
     console.log(`${verb} ${count} screenshot${plural}`);
-    if (count > 0) {
```

Comment:
What output was this responsible for?

Reply:
This gets called at the beginning of the test. It prints out the list of files that will be compared, added/removed, and skipped.

I was trying to clean up some log spam, but I'll revert this change for now so we can get the GitHub fix in.